.github/workflows/wkdev-sdk.yml

```diff
@@ -31,12 +29,13 @@ jobs:
           source ./register-sdk-on-host.sh
           wkdev-sdk-bakery --mode=build --verbose
           wkdev-sdk-bakery --mode=export --verbose
+          mv wkdev-sdk.tar wkdev-sdk-amd64.tar
```

Comment:
Update the export script to use the correct naming.

Reply:
We could add a `--arch` option to `wkdev-sdk-bakery`?